storage/Filesystems/BtrfsImpl.cc

```diff
@@ -88,6 +113,28 @@ namespace storage
     }
 
 
+    FilesystemUser*
+    Btrfs::Impl::add_device(BlkDevice* blk_device)
+    {
+	if (blk_device->num_children() != 0)
```

Comment:
I guess this check will stay like this only during a transitional period, right?
In the future, it will need to check if the resulting number of children matches the BtrfsRaidLevel.
Like this, it only works with one single block device.

Reply:
This check ensures that the block device being added has no child yet, so is unused.